job/job.go

```diff
@@ -150,13 +161,6 @@ func (j *Job) Init(cache JobCache) error {
 	j.lock.Lock()
 	defer j.lock.Unlock()
 
-	// Job Validation
-	// TODO: Move this to a seperated method?
-	if j.Name == "" || j.Command == "" {
-		log.Errorf(ErrInvalidJob.Error())
-		return ErrInvalidJob
-	}
-
```

Comment:
Shouldn't `j.validation()` be called here instead? Or is it being called elsewhere and I just don't see it?